### PR TITLE
GrafanaLabs.GrafanaOSS version 8.0.2.0

### PR DESCRIPTION
**Pull request by ImJoakim**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/17522)

### CODE REVIEW

File: manifests/g/GrafanaLabs/GrafanaOSS/8.0.2/GrafanaLabs.GrafanaOSS.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: GrafanaLabs.GrafanaOSS
+PackageVersion: 8.0.2
```
Comment (ItzLevvie2) [inline manifests/g/GrafanaLabs/GrafanaOSS/8.0.2/GrafanaLabs.GrafanaOSS.installer.yaml:4]:
```suggestion
PackageVersion: 8.0.2.0
```

File: manifests/g/GrafanaLabs/GrafanaOSS/8.0.2/GrafanaLabs.GrafanaOSS.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate
+
+PackageIdentifier: GrafanaLabs.GrafanaOSS
+PackageVersion: 8.0.2
```
Comment (ItzLevvie2) [inline manifests/g/GrafanaLabs/GrafanaOSS/8.0.2/GrafanaLabs.GrafanaOSS.locale.en-US.yaml:4]:
```suggestion
PackageVersion: 8.0.2.0
```

File: manifests/g/GrafanaLabs/GrafanaOSS/8.0.2/GrafanaLabs.GrafanaOSS.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate
+
+PackageIdentifier: GrafanaLabs.GrafanaOSS
+PackageVersion: 8.0.2
+PackageLocale: en-US
+Publisher: Grafana Labs
+PackageName: Grafana Open Source
```
Comment (ItzLevvie2) [inline manifests/g/GrafanaLabs/GrafanaOSS/8.0.2/GrafanaLabs.GrafanaOSS.locale.en-US.yaml:7]:
```suggestion
PackageName: GrafanaOSS
```

File: manifests/g/GrafanaLabs/GrafanaOSS/8.0.2/GrafanaLabs.GrafanaOSS.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: GrafanaLabs.GrafanaOSS
+PackageVersion: 8.0.2
```
Comment (ItzLevvie2) [inline manifests/g/GrafanaLabs/GrafanaOSS/8.0.2/GrafanaLabs.GrafanaOSS.yaml:4]:
```suggestion
PackageVersion: 8.0.2.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please see my review.

**Comment by ItzLevvie** [comment]
You'll have to update the version folder as well.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**